ossdbtoolsservice/language/completion/mysqlcompleter.py

```diff
@@ -0,0 +1,991 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from typing import List
```

Comment:
sort them alphabetically and put import statements first and then from statements